essentials/src/api/subxt_wrapper.rs

```diff
@@ -517,20 +519,26 @@ fn decode_extrinsic(data: &mut &[u8]) -> std::result::Result<SubxtCall, DecodeEx
 }
 
 async fn subxt_get_sheduled_paras(api: &OnlineClient<PolkadotConfig>, block_hash: H256) -> Result {
-	let addr = polkadot::storage().para_scheduler().scheduled();
-	let scheduled_paras = api.storage().at(block_hash).fetch(&addr).await?.unwrap_or_default();
-	Ok(Response::ScheduledParas(scheduled_paras))
+	let addr = subxt::dynamic::storage_root("ParaScheduler", "Scheduled");
+	let value = api.storage().at(block_hash).fetch(&addr).await?.unwrap().to_value()?;
```

Comment:
nit: `.await?.unwrap().to_value()?;` the `unwrap` here and similar below could fail if we stop providing the storage entry, would except make it more explicit?

Reply:
If we return a Result, it might be even worth to convert that to a non-fatal error (especially for the mode with running on all heads)

Reply:
Done